### PR TITLE
Feature/date inputs

### PR DESCRIPTION
**Pull request by apercas**
image
Added date inputs that change the visible timespan.

### CODE REVIEW

File: app/src/components/map.jsx [hunk 1]
```diff
@@ -6,8 +6,8 @@ import createOverlayLayer from './layers/vesselOneLayer';
 import CanvasLayer from './layers/canvasLayer';
 import map from '../../styles/index.scss';
 
-const min2015 = 1420070400000; // 1/1/2015
-const max2015 = 1451606400000; // 1/1/2015
+let tmlnMinDate = 1420070400000; // 1/1/2015
+let tmlnMaxDate = 1451606400000; // 1/1/2015
```
Comment (tiagojsag) [inline app/src/components/map.jsx]: guess you mean 2016

File: app/src/components/map.jsx [hunk 2]
```diff
@@ -39,19 +39,19 @@ class Map extends Component {
...
     if (!this.state.running) return;
 
     ite = ite || 0;
-    if (ite > max2015) {
+    if (ite > tmlnMaxDate) {
       this.setState({running: !!!this.state.running, ite: null, widthRange: 0});
       return ;
     }
-    let width = ((ite - min2015) / mDay)/ ((max2015 - min2015) / mDay) * 100;
+    let width = ((ite - tmlnMinDate) / mDay)/ ((tmlnMaxDate - tmlnMinDate) / mDay) * 100;
```
Comment (tiagojsag) [inline app/src/components/map.jsx]: not for now, but in the future, please use more human-friendly variable names. minifiers can shorten them later on, and when you start operating over them like this, it's easy to lose track of what's really going on.

**Comment by apercas** [comment]
Whoops, yes, the max date is 1st JAN 2016.

**Comment by davidsingal** [comment]
I've tested and it doesn't update timeline.

**Comment by davidsingal** [comment]
Good job! I've tested again and it works better now

**Comment by davidsingal** [comment]
For me this PR can be merged.

**Pull request merged**